### DISCUSSION TITLE
Community strength vs professional services

### DISCUSSION THREAD

**Discussion by ICLH** [keycloak/keycloak#15787] [Q&A]
Good morning Keycloak community! Our organisation is thinking about moving to Keycloak as an open-source SSO solution for CIAM, but there’s understandably some apprehension about going for this over a paid-for-supported service – especially given that we have all kinds of app-related issues that we’ll have to overcome to get it successfully implemented (it's likely to be a long and complex process!). 

To help allay our manager’s fears, I’d be interested to hear your experiences of how this community has helped you to solve various Keycloak questions / problems. Have you almost always found the answers that you needed online? Have you ever had to revert to professional services to help? If so, do you have any experiences of UK based consultancies? 

Any and all experiences welcomed. I know that it’s a very vague question and everyone’s Keycloak experiences will be different, I’m just trying to judge the level of support out there. Thanks!

**Comment by sschu** [comment]
There is a commercial version of Keycloak called Red Hat SSO that is professionally supported by Red Hat. Have looked at this?

**Reply by ICLH** [reply]
Thanks Sebastian, yes we're considering exactly this alongside looking at the community support if we were to go on our own.
